### ISSUE TITLE
Conda alias doesn't display conda help

### ISSUE THREAD

**Issue by mangecoeur** [gforsyth/xonda#9]
Typing `conda` in bash with no arguments displays the conda help, with `xonda` I get:

```
$ conda

<module 'conda' from '~/miniconda3/lib/python3.6/site-packages/conda/__init__.py'
```

**Comment by gforsyth** [comment]
Thanks for reporting, @mangecoeur -- this is an interesting one... 
We import `conda.install` in order to create the necessary symlinks in new environments, and since xonsh is a python instance, we can't 'unload' that import to make it default to the help screen.

Ugly possibly solutions include monkeypatching the repr...

**Comment by gforsyth** [comment]
Well, that was simpler to fix than I thought -- just renamed the library on import and no more issue.  

Fixed in 6e5710a40da2
